Examples/MAX32650/OTP_Dump/main.c

```diff
@@ -0,0 +1,164 @@
+/******************************************************************************
+ * Copyright (C) 2023 Maxim Integrated Products, Inc., All rights Reserved.
+ * 
+ * This software is protected by copyright laws of the United States and
+ * of foreign countries. This material may also be protected by patent laws
+ * and technology transfer regulations of the United States and of foreign
+ * countries. This software is furnished under a license agreement and/or a
+ * nondisclosure agreement and may only be used or reproduced in accordance
+ * with the terms of those agreements. Dissemination of this information to
+ * any party or parties not specified in the license agreement and/or
+ * nondisclosure agreement is expressly prohibited.
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL MAXIM INTEGRATED BE LIABLE FOR ANY CLAIM, DAMAGES
+ * OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * Except as contained in this notice, the name of Maxim Integrated
+ * Products, Inc. shall not be used except as stated in the Maxim Integrated
+ * Products, Inc. Branding Policy.
+ *
+ * The mere transfer of this software does not imply any licenses
+ * of trade secrets, proprietary technology, copyrights, patents,
+ * trademarks, maskwork rights, or any other form of intellectual
+ * property whatsoever. Maxim Integrated Products, Inc. retains all
+ * ownership rights.
+ *
+ ******************************************************************************/
+
+/**
+ * @file    main.c
+ * @brief   OTP Dump Example
+ * @details This example demonstrates how to read/write data from OTP memory
+ *
+ */
+
+/***** Includes *****/
+#include <stdio.h>
+#include <string.h>
+#include <stdint.h>
+
+#include "flc.h"
+#include "mxc_device.h"
+
+/***** Definitions *****/
+/*
+ *  Attention if you set WITH_WRITE_TEST flag
+ *  This example will search OTP and try to find 0xffffffff data
+ *  If it could be found 4 test bytes will be written in that address.
+ */
+#define WITH_WRITE_TEST 0 // set it to test write test
+
+// OTP Address
+#define OTP_MAXIM_AREA MXC_INFO_MEM_BASE
+#define OTP_MAXIM_AREA_SIZE 0x1000
```

Comment:
Ditto

"Update to "ADI AREA" instead? If applicable, we should probably start replacing the "Maxim" names with "ADI" or something similar."